sydent/http/federation_tls_options.py

```diff
@@ -89,9 +89,8 @@ class ClientTLSOptionsFactory(object):
     """Factory for Twisted ClientTLSOptions that are used to make connections
     to remote servers for federation."""
 
-    def __init__(self):
-        # We don't use config options yet
-        self._options = ssl.CertificateOptions(verify=False)
+    def __init__(self, config):
+        self._options = ssl.CertificateOptions(verify=config.get("http", "federation.verifycerts"))
```

Comment:
Are you sure this does what you want? `"False"` is a non-empty string...

(hint: https://docs.python.org/2.7/library/configparser.html#ConfigParser.RawConfigParser.getboolean)